handler.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var craigslistApartments = require('./craigslist-apartments');
+var filters = require('./filters');
+const Filter = new filters();
+
+module.exports.hello =   (event, context) => {
+  var apartmentsPromise = craigslistApartments.getListOfRentals('0',[]);
+  return apartmentsPromise.then((result) => {
+    return result;
+  })
+  .then((result) => {
+    const testando = {
```

Comment:
Please use good variable names (in english).

In this case, you may also want to add a comment stating that this variable/lambda output is temporary.